tests/test_zarr.py

```diff
@@ -22,6 +22,90 @@ def runtime():
     yield acquire.Runtime()
 
 
+def test_set_acquisition_dimensions(
+    runtime: Runtime, request: pytest.FixtureRequest
+):
+    dm = runtime.device_manager()
+    props = runtime.get_configuration()
+    props.video[0].camera.identifier = dm.select(
+        DeviceKind.Camera, ".*empty.*"
+    )
+    props.video[0].camera.settings.shape = (64, 48)
+
+    props.video[0].storage.identifier = dm.select(DeviceKind.Storage, "Zarr")
+    props.video[0].storage.settings.filename = f"{request.node.name}.zarr"
+    props.video[0].max_frame_count = 32
+
+    # configure storage dimensions
+    dimension_x = acquire.StorageDimension()
+    dimension_x.name = "x"
+    dimension_x.kind = acquire.DimensionType.Space
+    dimension_x.array_size_px = 64
+    dimension_x.chunk_size_px = 64
```

Comment:
Does this construction work?

```suggestion
    dimension_x = acquire.StorageDimension(name="x",
       kind=acquire.DimensionType.Space,
       array_size_px=64,
       chunk_size_px=64)
```

Also, we might support string translation for enums, 

```suggestion
    dimension_x = acquire.StorageDimension(name="x",
       kind="Space",
       array_size_px=64,
       chunk_size_px=64)
```

I'm not sure which is best, but I like using the constructor here.

Reply:
The former construction doesn't work (but I think I could figure out how to make it work). I don't like string translation for the enums. The values are already there and AFAIK we don't do it for the other enums.